src/styles/items.styl

```diff
@@ -61,6 +61,10 @@
                     margin: 0
                     padding-top: 5px
 
+            .faultHistory
```

Comment:
Nit: this one (`#items .item .infoArea .faultHistory`) is not necessary because the one below is `.item .infoArea .faultHistory`